### ISSUE TITLE
Cannot go back to main page from Spanish page

### ISSUE THREAD

**Issue by ManonGros** [gbif/hp-grscicoll#31]
**Describe the bug**
If I am on a Spanish page and try to click on the main GRSciColl upper left corner button (which leads me to this https://grscicoll.hp.gbif-staging.org/es), I get a 

```
Forbidden
You don't have permission to access /es on this server.
```

**To Reproduce**
1. Go to https://grscicoll.hp.gbif-staging.org/es/contact/
2. Click on the upper right corner GRSciColl button to go back to the home page

**Expected behavior**
I expect it to redirect me to the English website since there is no Spanish homepage

**Desktop (please complete the following information):**
 - MacOS Ventura 13.5.2 
 - Safari
 - Version 16.6

**Comment by vechocho** [comment]
I Think this is because there is not a spanish page i just made a pull request for this also I add the spanish in the _conf.yml.

Im using this hp as example https://github.com/gbif/hp-canadensys/tree/master

**Comment by MortenHofft** [comment]
I suspect it is a minimum that you translate the homepage if you want a multilingual site

**Comment by vechocho** [comment]
I couldnt translate the json file for the collections as is too large but im on it.

**Comment by MortenHofft** [comment]
technically it doesn't have to BE translated, it just need to exist. And we can translate the header area and leave the rest be or translate it as we go along.

**Comment by vechocho** [comment]
I aadded this 

` - scope:
      path: "es/" # For anything in the "es" folder the default is to set the lang frontMatter property to "es" . this should be deleted or adapted to your site if you support other languages
    values:
      lang: es
`
Is already on the _conf.yml pull request

**Comment by ManonGros** [comment]
Thank you @vechocho! It is so impressive to have those pages translated! I really appreciate.
 I had to change back the title to GRSciCOll on the homepage. I checked with our communication's team and we are worried that because the Spanish version shows a different acronym, it might get a bit confusing for the audience. In other places on the website, I think it would be perfectly fine to translate the name though.
